pkg/libs/endpointaccessible/endpoint_accessible_controller.go

```diff
@@ -54,7 +57,9 @@ func NewEndpointAccessibleController(
 		WithSync(c.sync).
 		ResyncEvery(wait.Jitter(time.Minute, 1.0)).
 		WithSyncDegradedOnError(operatorClient).
-		ToController(controllerName, recorder.WithComponentSuffix(name+"endpoint-accessible-controller"))
+		ToController(
+			controllerName, // Don't change what is passed here unless you also remove the old FooDegraded condition
```

Comment:
This one is a bit strange. Degraded is prefixed by `controllerName`, but Available is prefixed by `name`.

Reply:
Yes, but we cannot change these names I think.